.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+      - name: Build package
+        run: |
+          python setup.py compile_catalog sdist bdist_wheel
+      - name: pypi-publish
+        uses: pypa/gh-action-pypi-publish@v1.3.1
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_token }}
```

Comment:
Just after you left for vacation there was an issue where additional manual steps were required on GitHub to make this work (something to do with the token and its scope I think). Did you do those already?

Reply:
I'm not aware of said troubles 😅 *before* I left there was the need of setting it up, but it was set up and it worked for `docker-services-cli` already. Is this what you meant?

Reply:
It was a naming issue, since in the workflow the token was named `pypi_password`, while in the org settings it was `pypi_token`... Now both exist, but we should align to use one (I prefer `pypi_token`)